flink/pom.xml

```diff
@@ -97,44 +95,44 @@
 
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-actor_${flink.scala.binary.version}</artifactId>
+      <artifactId>akka-actor_${scala.binary.version}</artifactId>
```

Comment:
there might still be a need for interpreter individually requiring different version of Scala? I think for now it might make sense to have separate `flink.scala`, `ignite.scala` and so on